build/uri-def.py

```diff
@@ -396,7 +396,7 @@ def expand_prefix(txt, prefixes):
             
             if g7[tag][0] in ('structure', 'enumeration', 'calendar', 'month'):
                 ptag = re.sub(r'.*-', '', re.sub(r'-[A-Z]?[a-z].*', '', tag))
-                print('\nstandard tag: '+(repr(ptag) if ptag in ('YES','NO','TRUE','FALSE', '0', '1', '2', '3') else ptag), file=fh)
+                print('\nstandard tag: '+(repr(ptag) if ptag in ('TRUE','FALSE', '0', '1', '2', '3') else ptag), file=fh)
```

Comment:
We cannot remove quotes from strings that are also ways of spelling Boolean values without changing semantics. This suggested change breaks YAML.

If we want to be consistent in where quotes appear, we could instead make an unconditional use of `repr`